core/src/test/java/org/infinispan/distexec/ReplSyncDistributedExecutorTest.java

```diff
@@ -22,10 +22,4 @@ protected String cacheName() {
    protected CacheMode getCacheMode() {
       return CacheMode.REPL_SYNC;
    }
-
-   @Test(groups = "unstable")
-   @Override
-   public void testBasicTargetRemoteDistributedCallableWithHighFutureAndLowTaskTimeout() throws Exception {
```

Comment:
Well, you https://issues.jboss.org/browse/ISPN-3992 as out of date one year ago ;)

I'm pretty sure the test was fixed when you changed the return type to `CompletableFuture`, I just enabled it to make sure. The failures in the unstable tests build are irrelevant, because it didn't have the proper `expectedExceptions` annotation.